web/templates/settings/user_list_admin.hbs

```diff
@@ -1,27 +1,17 @@
-<div id="admin-user-list" class="settings-section" data-name="user-list-admin">
+<div id="users-admin-panel" class="settings-section" data-name="user-panel-admin">
 
-    <div class="clear-float"></div>
+    <div class="tab-container"></div>
 
-    <div class="settings_panel_list_header">
-        <h3>{{t "Users"}}</h3>
-        <div class="alert-notification" id="user-field-status"></div>
-        <input type="text" class="search filter_text_input" placeholder="{{t 'Filter users' }}" aria-label="{{t 'Filter users' }}"/>
+    <div class="user-section-container" data-user-settings-section="user-list-admin">
+        {{>active_user_list_admin}}
     </div>
 
-    <div class="progressive-table-wrapper" data-simplebar>
-        <table class="table table-striped wrapped-table">
-            <thead class="table-sticky-headers">
-                <th class="active" data-sort="alphabetic" data-sort-prop="full_name">{{t "Name" }}</th>
-                <th data-sort="email">{{t "Email" }}</th>
-                <th class="user_role" data-sort="role">{{t "Role" }}</th>
-                <th class="last_active" data-sort="last_active">{{t "Last active" }}</th>
-                {{#if is_admin}}
-                <th class="actions">{{t "Actions" }}</th>
-                {{/if}}
-            </thead>
-            <tbody id="admin_users_table" class="admin_user_table"
-              data-empty="{{t 'No users match your current filter.' }}"></tbody>
-        </table>
+    <div class="user-section-container" data-user-settings-section="deactivated-users-admin">
```

Comment:
I have removed the extra div. But the ```settings-section``` from each template file couldn't be removed because I need it to hide the users section in toggler callback.